app/permissions_migrations.go

```diff
@@ -74,18 +75,67 @@ const (
 	PermissionManageRemoteClusters           = "manage_remote_clusters" // deprecated; use `manage_secure_connections`
 )
 
+// Warning: This function should only be used in cases where team and channel scheme roles definitely do not need to be migrated.
```

Comment:
It would be nice if we could actually get rid of this or replace it with the including schemes version. It might be possible?

Reply:
There are 14 invocations of the old logic that I wouldn't want to update because they've either [been identified as not needing scheme role updates](https://github.com/mattermost/mattermost-server/pull/10590#discussion_r389087298) or 
would require the migration to be re-written and re-tested. 

There are 22 instances that I think could safely use the new logic. 

One option could be to rename the *old* function to something like `isExactRole` and mark it deprecated and name the new function `isRole`. That wouldn't remove it from code completely but it might increase the odds that people use the proper function in the future.

Reply:
Renaming would be perfect. I don't think we want to open up the can of old migration worms.

Reply:
Done. Let me know if you think that looks good.

Reply:
Looks good!